internal/ignition/ignition_test.go

```diff
@@ -740,6 +741,28 @@ var _ = Describe("downloadManifest", func() {
 	})
 })
 
+var _ = Describe("ParseTo32", func() {
+	const v32ignition = `{"ignition": {"version": "3.2.0"},"storage": {"files": []}}`
+	const v31ignition = `{"ignition": {"version": "3.1.0"},"storage": {"files": []}}`
+	It("parses a v32 config", func() {
+		config, err := ParseTo32([]byte(v32ignition))
+		Expect(err).ToNot(HaveOccurred())
+		Expect(config.Ignition.Version).To(Equal("3.2.0"))
+	})
+
+	It("parses a v31 config", func() {
+		config, err := ParseTo32([]byte(v31ignition))
+		Expect(err).ToNot(HaveOccurred())
+		Expect(config.Ignition.Version).To(Equal("3.1.0"))
+
+		bytes, err := json.Marshal(config)
+		Expect(err).ToNot(HaveOccurred())
+		v31Config, _, err := config_31.Parse(bytes)
+		Expect(err).ToNot(HaveOccurred())
+		Expect(v31Config.Ignition.Version).To(Equal("3.1.0"))
+	})
+})
+
```

Comment:
These tests do a pretty good job of describing what we're trying to accomplish generally.